src/stack_height/max_height.rs

```diff
@@ -410,6 +410,13 @@ pub(crate) fn compute(func_idx: u32, module: &elements::Module) -> Result<u32, E
 				stack.pop_values(1)?;
 				stack.push_values(1)?;
 			},
+
+			#[cfg(feature = "sign_ext")]
+			SignExt(_) => {
+				stack.pop_values(1)?;
+				stack.push_values(1)?;
+			}
```

Comment:
I am a bit worried about this, in case there are more instructions added to this enum. Would it be too much hassle to explicitly list these instructions here?

Reply:
@pepyakin Sure, done.